support/setup-dev.sh

```diff
@@ -27,7 +27,7 @@ __EOF__
 fi
 
 if test ! -e .gitignore; then
```

Comment:
Maybe add a comment explaining why a copy is necessary, pointing to https://github.com/git/git/blob/master/Documentation/RelNotes/2.32.0.txt ?

Reply:
:+1: